release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3340,6 +3340,59 @@ This section will continue to be updated over time to provide notes on enhanceme
+=== RHSA-2024:6406 - {product-title} 4.14.36 bug fix and security update
+
+Issued: 11 September 2024
+
+{product-title} release 4.14.36, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6406[RHSA-2024:6406] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6412[RHSA-2024:6412] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.36 --pullspecs
+----
+
+[id="ocp-4-14-36-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-16-36-enhancements-_{context}"]
+===== Updating the CENTOS 8 references to CENTOS 9
+
+* CENTOS 8 has recently ended its lifecycle. This release updates the CENTOS 8 references to CENTOS 9.
+(link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*])
+
+[id="ocp-4-14-36-enhancements-collecting-ingress-controller-certificates_{context}"]
+===== Collecting Ingress Controller certificates
+
+* The Insights Operator now collects information about all Ingress Controller certificates (NotBefore and NotAfter dates). It aggregates it into a JSON file in the path 'aggregated/ingress_controllers_certs.json'. (link:https://issues.redhat.com/browse/OCPBUGS-37673[*OCPBUGS-37673*])
+
+[id="ocp-4-14-36-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, if a virtual machine (VM) was deleted and the network interface controller (NIC) still existed for that VM, the {azure-first} VM verification check crashed. With this release, the verification check can now handle this situation by gracefully processing the issue without crashing. (link:https://issues.redhat.com/browse/OCPBUGS-39413[*OCPBUGS-39413*])
+
+* Previously, the `spec.noProxy` field from the cluster-wide proxy was not considered when the {cmo-first} configured proxy capabilities for a Prometheus remote write endpoint. With this release, the {cmo-short} no longer configures proxy capabilities for any remote write endpoints whose URL would bypass the proxy according to the `noProxy` field. (link:https://issues.redhat.com/browse/OCPBUGS-39176[*OCPBUGS-39176*])
+
+* Previously, when running `oc logs -f <pod>`, the logs would not output anything after the log file was rotated. With this release, kubelet will output a log file after it has been rotated and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38959[*OCPBUGS-38959*])
+
+* Previously, the {product-title} web console failed to restart a bare metal node. This release fixes this issue so that you can restart a bare metal node by using the {product-title} web console. (link:https://issues.redhat.com/browse/OCPBUGS-38053[*OCPBUGS-38053*])
```

Comment:
```suggestion
* Previously, the {product-title} web console failed to restart a bare-metal node. This release fixes this issue so that you can restart a bare metal node by using the {product-title} web console. (link:https://issues.redhat.com/browse/OCPBUGS-38053[*OCPBUGS-38053*])
```